nl/guides/for-students/login-and-settings/README.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Aanmelden & gebruikersinstellingen
+description: "Tutorial: aanmelden en gebruikersinstellingen voor studenten"
+next: ../exercises/
+---
+
+## Selecteren van de taal
+
+Als **taal** voor Dodona kan je kiezen tussen Nederlands of Engels. Gebruik hiervoor het **taalmenu** aan de rechterkant van de **navigatiebalk** (bovenaan elke pagina).
+
+![image](./choose_language.png)
+
+Als je een [taal](#selecteren-van-de-taal) selecteert terwijl je [aangemeld](#aanmelden) bent dan wordt die ook als voorkeurstaal ingesteld in je [gebruikersprofiel](#instellen-van-persoonlijke-voorkeuren). Op die manier wordt bij aanmelden automatisch je voorkeurstaal geselecteerd.
+
+::: details Notitie
+
+De taal waarin vaste componenten van Dodona (menubalken, cursusstructuur, mededelingen, ...) worden weergegeven, valt volledig onder de controle van het platform. Deze componenten worden altijd weergegeven in de geselecteerde [taal](#selecteren-van-de-taal).
+
+De naam en beschrijving van een [cursus]() liggen vast, net zoals namen en beschrijvingen van [oefeningenreeksen]() binnen een cursus. Deze componenten zijn niet afhankelijk van de selectie van de [taal](#selecteren-van-de-taal). De taal van deze componenten valt volledig onder de controle van de cursusbeheerders die de cursus hebben opgesteld.
+
+De naam en beschrijving van een [oefening](../exercises/#navigeren-naar-een-oefening) zijn afhankelijk van de selectie van de [taal](#selecteren-van-de-taal). Als bij het opstellen van de oefening een vertaling voorzien werd van de naam en de beschrijving in de geselecteerde taal, dan zullen deze componenten van de oefening ook in die taal weergegeven worden.
+
+De [taal](#selecteren-van-de-taal) van de [feedback](../exercises/#interpreteren-van-feedback) over een [oplossing](../exercises/#navigeren-naar-een-oplossing) die je [indient](../exercises/#indienen-van-een-oplossing) voor een [oefening](../exercises/#navigeren-naar-een-oefening) valt volledig onder de controle van de [judge](../exercises/#interpreteren-van-feedback) waaraan de oefening gekoppeld is. De judge kan de taal van deze feedback laten afhangen van de taal die geselecteerd is op het ogenblik waarop de oplossing wordt ingediend.
+:::
+
+## Aanmelden
+
+Om te kunnen [aanmelden](#aanmelden), moet je over een **gebruikersaccount** beschikken op Dodona. In het daaraan gekoppelde [gebruikersprofiel](#instellen-van-persoonlijke-voorkeuren) kan je persoonlijke voorkeuren instellen om je gebruikerservaring op het platform te personaliseren. Daarnaast worden alle acties die je op het platform uitvoert &mdash; zoals het [registreren](../courses/#registreren-voor-een-cursus) voor [cursussen]() of het [indienen](../exercises/#indienen-van-een-oplossing) van [oplossingen](../exercises/#navigeren-naar-een-oplossing) &mdash; aan je gebruikersaccount gekoppeld.
```

Comment:
The first line of this paragraph is very confusing and misses the point. This section should start with the fact that an account is automatically created when you first sign in (right now this is mentioned only several paragraphs below) and that you can only sign in using an account of your school (which is in a "tip" but should be in the action text). Again, remove useless links.